modules/features/player/src/main/java/au/com/shiftyjelly/pocketcasts/player/view/ShelfItem.kt

```diff
@@ -81,4 +81,21 @@ sealed class ShelfItem(val id: String, var title: (Playable?) -> Int, var iconRe
         },
         Shown.Always
     )
+
+    enum class ShelfItemId(val value: String, val analyticsValue: String) {
+        EFFECTS("effects", "playback_effects"),
+        SLEEP("sleep", "sleep_timer"),
+        STAR("star", "star_episode"),
+        SHARE("share", "share_episode"),
+        PODCAST("podcast", "go_to_podcast"),
+        CAST("cast", "chromecast"),
+        PLAYED("played", "mark_as_played"),
+        ARCHIVE("archive", "archive"),
+        DOWNLOAD("download", "download");
+
+        companion object {
+            fun fromId(id: String?) =
+                ShelfItemId.values().find { it.value == id }
+        }
+    }
```

Comment:
Instead of having this separate `ShelfItemId` class that we use to map from ids to analytics values, what would you think about adding the analytics value to the `ShelfItem` class itself?

I tried this out to see if it worked, and it seemed to. You can take a look at what I mean [here](https://github.com/Automattic/pocket-casts-android/commit/8b3145ffa159345cac5cebd5b639c7e248634c6f).

Reply:
That looks better! Thanks for the commit, I've cherry-picked it.